src/app.module.ts

```diff
@@ -12,6 +13,6 @@ import { AppController } from "./app.controller";
     MongooseModule.forRoot(process.env.MONGODB_URL)
   ],
   controllers: [AppController],
-  providers: []
+  providers: [MailService]
```

Comment:
Mail stuff should probably be its own module rather than just a service (for the dependency injection reasons and also to keep in style with the other modules).